tests/trx_finality_status_forked_test.py

```diff
@@ -228,11 +229,16 @@ def getBlockID(status):
         f"ERROR: Block never finalized.\n\nprod A info: {json.dumps(prodA.getInfo(), indent=1)}\n\nprod C info: {json.dumps(prodD.getInfo(), indent=1)}" + \
         f"\n\nafter fork in block state: {json.dumps(afterForkInBlockState, indent=1)}"
 
-    retStatus = prodD.getTransactionStatus(transId)
-    if afterForkBlockId != getBlockID(retStatus): # might have been forked out, if so wait for new block to become LIB
-        assert prodD.waitForBlock(getBlockNum(retStatus), timeout=120, blockType=BlockType.lib), \
+    while True: # might have been forked out, if so wait for new block to become LIB
+        retStatus = prodD.getTransactionStatus(transId)
+        currentBlockId = getBlockID(retStatus)
+        currentBlockNum = getBlockNum(retStatus)
+        if afterForkBlockId == currentBlockId:
+            break
+        assert prodD.waitForBlock(currentBlockNum, timeout=120, blockType=BlockType.lib), \
```

Comment:
I can't tell.. is this one of those things we want to avoid (#332)?

Reply:
Yes, according to #332, but it is pervasive in our tests. I think when #332 is worked we need to find an easy to use alternative. Likely we will just go with: `if not blah: exitError("blah")` but we might add our own `assert` function.